docker-compose.yml

```diff
@@ -28,7 +27,6 @@ services:
       TEST_OPENSEARCH_PORT: 9200
       TEST_POSTGRES_HOST: postgres
       TEST_PRESTO_HOST: presto
-      TEST_PRESTO_PORT: 8080
```

Comment:
should this line be removed? The env var is still referenced [here](https://github.com/DataDog/dd-trace-rb/blob/7622101888c86f1ab49cb95a714af19e5c44a506/docker-compose.yml#L290).

Reply:
I think so, it is provided by `.env` file. 

For each service, we are applying environment variables by providing `env_file` and overwrite with environment. 

For example:

```
    env_file: ./.env
    environment: 
      -  ...
```

Reply:
thanks! I missed the `.env` file